imports/eosio.imports.in

```diff
@@ -192,3 +192,5 @@ is_provided_bw_confirmed
 set_bw_limits
 confirm_bw_limits
 get_ram_provider
+update_stake_proxied
+recall_stake_proxied
```

Comment:
it's better to add some C and/or C++ interface to be able to call this functions (like https://github.com/GolosChain/cyberway.cdt/pull/19/files)

Reply:
`privileged.h` is good place for that interface